src/main/java/model/ladder/Line.java

```diff
@@ -0,0 +1,61 @@
+package model.ladder;
+
+import model.player.Players;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    private static final int START_POINT = 0;
+    private static final int DEFAULT_INTERVAL = 5;
+    private List<Point> points = new ArrayList<>();
+
+    public static Line of(Players players) {
+        final Line line = new Line();
+        line.createLineByPlayers((players.getPlayerCount() - 1), START_POINT);
+        return line;
+    }
+
+    private void createLineByPlayers(final int totalPoint, final int currentPoint){
+
+        if(totalPoint == currentPoint){
+            points.add(new Point(Point.State.BAR));
+            return;
+        }
+
+        points.add(new Point(Point.State.BAR));
+
+        if(isBeforeLine()){
+            createIntervalByPoint(Point.State.SPACE);
+            createLineByPlayers(totalPoint, (currentPoint + 1));
+            return;
+        }
+
+        createIntervalByPoint(Point.generateStateByRandom());
+        createLineByPlayers(totalPoint, (currentPoint + 1));
```

Comment:
이전에 bar가 존재하는지 확인해서 SPACE로 처리하시는 거라면, 아래처럼 하는게 중복 제거하는면에서 더 좋을거 같아요
```suggestion
        Point.State state = Point.generateStateByRandom();
        if(isBeforeLine()){
            state = Point.State.SPACE;
        }
        
        createIntervalByPoint(state);
        createLineByPlayers(totalPoint, (currentPoint + 1));
```

Reply:
넵 저 부분은 좀 더 가독성 있게 수정하는 것이 더 좋겠어요 :) !